src/coreclr/vm/interpexec.cpp

```diff
@@ -88,6 +120,7 @@ void InvokeUnmanagedMethodWithTransition(MethodDesc *targetMethod, int8_t *stack
         PCODE callTarget;
     }
     param = { targetMethod, pArgs, pRet, callTarget };
+#endif // TARGET_ARM
 
     PAL_TRY(Param *, pParam, &param)
```

Comment:
```suggestion
    PAL_TRY(UnmanagedMethodWithTransitionParam *, pParam, pParam)
```
And then use the same structure as the parameter for PAL_TRY macro